### ISSUE TITLE
dash-to dock, can't install it on fedora 36 WS

### ISSUE THREAD

**Issue by hansgsenger** [micheleg/dash-to-dock#1782]
Hello,

can't install it,  Fedora 36 Workstation, X11, GNOME 42,3  made "export SASS=dart" befor. The error message is: 

[elias@ryzen9 Downloads]$ make -C dash-to-dock install
make: directory "/home/elias/Downloads/dash-to-dock" is entered
sass --no-source-map _stylesheet.scss stylesheet.css
make: sass: file or directory not found
make: *** [Makefile:62: stylesheet.css] error 127
make: directory "/home/elias/Downloads/dash-to-dock" is left

Thank you

**Comment by hansgsenger** [comment]
Solved
first I tried  to install  sass with

sudo npm install -g sass (bash asked me if it should install npm, what I agreed to)
sudo npm install -g npm@8.15.1  was a suggestion of bash to update sass

then i had to do

export SASS=dart       a trap as i realised, as it gave back on echo $? a Zero means  success, but it wasn't, with

sudo export SASS=dart      I got the error message "that the command export would be unknown"! As it is not part defined in $PATH, where dnf is  looking for.   epxort is a kind of "sublevel command" of  bash  so I found out that i had to do

sudo bash -c "export SASS=dart"  and the command export was accepted  and finally  sass  installed properly

so last the 
step to victory
make -C dash-to-dock install       made my Das-to-Dock alive !

and it doesn't crash under some circumstances as Dash-to-Dock for COSMIC does.

.P.S: in the Browser it is not possible to activate D-t-D

but with “Extension Manager” from the Softwarecenter  it is recognised!!
